src/app/pages/datasets/components/dataset-management/dataset-management.component.html

```diff
@@ -86,6 +87,7 @@
                 routerLinkActive="selected"
                 [routerLink]="['/datasets', dataset.id]"
                 (click)="openMobileDetails()"
+                (keydown.enter)="openMobileDetails(['/datasets', dataset.id])"
```

Comment:
It looks like you can unify all of this: 
1. Remove routerLink directive.
2. Have the same handler for (click) and (keydown.enter).
3. Call it something else like `onRowSelected(dataset: Dataset)`